test/deploy/rackhd_ha_install.py

```diff
@@ -0,0 +1,109 @@
+                             0, "Control network config failure.")
+            os.remove('control.cfg')
+            # startup NIC
+            fit_common.remote_shell('ip addr add ' + control_ip + '/22 dev ' + ifslist[1], vmnum=vmnum)
+            fit_common.remote_shell('ip link set ' + ifslist[1] + ' up', vmnum=vmnum)
+            self.assertEqual(fit_common.remote_shell('ping -c 1 -w 5 ' + control_ip, vmnum=vmnum)['exitcode'],
+                             0, 'Control NIC failure.')
+
+    def test02_install_dhcp_config(self):
+        # create DHCP config
+        for vmnum in range(1, numvms + 1):
+            fit_common.remote_shell('echo INTERFACES=' + ifslist[1] + ' > /etc/default/isc-dhcp-server', vmnum=vmnum)
+            dhcp_conf = open('dhcpd.conf', 'w')
+            dhcp_conf.write('ddns-update-style none;\n'
+                            'option domain-name "example.org";\n'
+                            'option domain-name-servers ns1.example.org, ns2.example.org;\n'
+                            'default-lease-time 600;\n'
+                            'max-lease-time 7200;\n'
+                            'log-facility local7;\n'
+                            'deny duplicates;\n'
+                            'ignore-client-uids true;\n'
+                            'subnet 172.31.128.0 netmask 255.255.252.0 {\n'
+                            '  range 172.31.128.100 172.31.131.254;\n'
+                            '  option vendor-class-identifier "PXEClient";\n'
+                            '}\n')
+            dhcp_conf.close()
+            # copy file to ORA
+            fit_common.scp_file_to_ora('dhcpd.conf', vmnum)
+            self.assertEqual(fit_common.remote_shell('cp dhcpd.conf /etc/dhcp/', vmnum=vmnum)['exitcode'],
+                             0, "DHCP Config failure.")
+            os.remove('dhcpd.conf')
+
+    def create_hosts(self):
+        hosts_conf = open('hosts-conf', 'w')
+        for vmnum in range(1, numvms + 1):
+            line = '172.31.128.{}\tnode{}\n'.format(vmnum, vmnum)
+            hosts_conf.write(line)
+        hosts_conf.close()
+
+    def test04_install_hosts_config(self):
+        for vmnum in range(1, numvms + 1):
+            self.create_hosts()
+            # copy file to ORA
+            fit_common.scp_file_to_ora('hosts-conf', vmnum)
+            self.assertEqual(fit_common.remote_shell('cat hosts-conf >> /etc/hosts', vmnum=vmnum)['exitcode'],
+                             0, "Hosts Config failure.")
+        os.remove('hosts-conf')
+
+    def test04_install_hostname_config(self):
+        for vmnum in range(1, numvms + 1):
+            command = "echo node{} > /etc/hostname".format(vmnum)
+            self.assertEqual(fit_common.remote_shell(command, vmnum=vmnum)['exitcode'], 0, "Hostname Config failure.")
+
+
+if __name__ == '__main__':
+    fit_common.unittest.main()
```

Comment:
can get rid of fit_common here  - ie. import unittest above and change to unittest.main